Basic CLI (#83)
Bouni/python-luxtronik: README.md

# check https://github.com/Bouni/luxtronik/blob/master/luxtronik/visibilities.py for values you might need
```

### COMMAND LINE INTERFACE

The luxtonik package provides several scripts that can be used to interact with your heatpump.

### DISCOVER

To discover all heatpumps in your network, you can use this CLI command:

```sh
luxtronik discover
```

It will send magic broadcast packages and all heatpumps will respond like this:

```sh
1 heatpump(s) reported back                                                                                                                                                                                      │
Heat pump #0 -> IP address: 192.168.178.123 port: 8889
```

> ### DUMP ALL DATA

Reviewer: ```suggestion
### DUMP ALL AVAILABLE DATA FROM THE HEAT PUMP
```

To get all data available you can either use the CLI:

```sh
luxtronik dump 192.168.178.123 8889
```

> or call the script that comes with the python package:

Reviewer: ```suggestion
Alternatively you can also invoke the script that is installed along with the Python package directly, e.g. invoke it in the following way:
```

Number: 11    Name: ID_Einst_HzHwHKE_akt                                         Type: Celsius              Value: 33.0                                                                                          │
Number: 12    Name: ID_Einst_HzHKRANH_akt                                        Type: Celsius              Value: 22.0                                                                                          │
Number: 13    Name: ID_Einst_HzHKRABS_akt                                        Type: Celsius              Value: 0.0                                                                                           │
Number: 14    Name: ID_Einst_HzMK1E_akt                                          Type: Unknown              Value: 330                                                                                           │
Number: 15    Name: ID_Einst_HzMK1ANH_akt                                        Type: Unknown              Value: 220                                                                                           │
Number: 16    Name: ID_Einst_HzMK1ABS_akt                                        Type: Unknown              Value: 0
...
```

> ### SHOW CHANGES

Reviewer: ```suggestion
### SHOW CHANGED VALUES ONLY
```

> 

Reviewer: ```suggestion

There is another sub-command (`changes`) and/or script (`dump-changes.py`) that will only show values that have recently changed. This is meant to be used interactively, i.e. the current value of specific settings will be shown. This is especially useful to identify yet unknown parameters. You can closely monitor the output while changing values on the Luxtronik controller itself.
```

> To get all values that change live diplayed you can either use the CLI:

Reviewer: ```suggestion
This can be invoked in the following ways:
```

```sh
luxtronik changes 192.168.178.123 8889
```

> or call the script that comes with the python package:

Reviewer: ```suggestion
Alternatively, it can be invoked directly:
```
